### ISSUE TITLE
dismissOnHardwareBackPress = false and dismissOnTouchOutside = false not working

### ISSUE THREAD

**Issue by boliveira** [jacklam718/react-native-dialog-component#2]
Hi,

I tried to show a dialog with DialogManager.show with dismissOnHardwareBackPress = false and dismissOnTouchOutside = false and it's not working, it's dismissing anyway.
Took a look at the code and the dismissOnHardwareBackPress is not passed to PopupDialog instance on DialogComponent render, and on Dialog.js there is a property closeOnTouchOutside that exists for backwards compatibility but it's value is by default true and it ignores the dismissOnTouchOutside value.

**Comment by jacklam718** [comment]
@boliveira
Did you set `haveOverlay` = false? because if haveOverlay is false then dismissOnTouchOutside won't work.

**Comment by boliveira** [comment]
No, haveOverlay is true, I checked that. I ended up changing Dialog.js code to remove the close*** props and changed DialogComponent.js to pass dismissOnHardwareBackPress to PopupDialog and then it worked.

**Comment by jacklam718** [comment]
@boliveira Can you try version `v0.1.10` thanks. if there are any problem please let me know. Thanks a lot.

**Comment by jacklam718** [comment]
@boliveira Can you try version `v0.1.11` thanks. if there are any problem please let me know. Thanks a lot.

**Comment by boliveira** [comment]
@jacklam718 , tested version `v0.1.11` and it works now. Thank you very much!
